src/app.jsx

```diff
@@ -16,16 +18,54 @@ import SettingsModal from './components/SettingsModal';
 import { getInvestModelNames } from './server_requests';
 import { getLogger } from './logger';
 import InvestJob from './InvestJob';
-import { dragOverHandlerNone } from './utils.js';
+import { dragOverHandlerNone } from './utils';
 
 const logger = getLogger(__filename.split('/').slice(-1)[0]);
 
+const investSettingsStore = localforage.createInstance({
+  name: 'InvestSettings',
+});
```

Comment:
Same question here about whether we always need to create a new instance since most of the time we expect there's already one in storage, right?

Reply:
Right! See my other reply. I just played around with using `localforage.config` in both `src` and `InvestJobs`, defining the `name` and `storeName` for each, but that did not seem to gel. It seems in that case you still need to `createInstance` otherwise `clear()` will wipe all keys regardless of the db you set up in `config`. So I think this `createInstance` is okay and in this case `investSettingsStore` will either be a clean slate or have the values we'd expect if it was loaded. But these instances seem to help separate the `clear()` calls so they only occur on those instances.

That might not have made a lot of sense, happy to chat more / Google more.